main.go

```diff
@@ -163,14 +192,13 @@ func mailHandler(peer smtpd.Peer, env smtpd.Envelope) error {
 		case "login":
 			auth = LoginAuth(*remoteUser, *remotePass)
 		default:
+			// TODO: log
```

Comment:
?

Reply:
Yep, forgot to come back to this. This is one case (of several I've come across) where we should really detect this right after loading the config, instead of while handling a connection.

Reply:
I didn't feel like pulling out that logic in this PR since it would be mostly duplicated. I just removed the TODO since there's more to be done in that area anyway.